client/state/comments/schema.js

```diff
@@ -0,0 +1,83 @@
+export const commentsSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+-\\d+$': {
+			type: 'array',
+			items: {
+				type: 'object',
+				properties: {
+					ID: { type: 'integer' },
+					URL: { type: 'string' },
+					author: {
+						type: 'object',
+						properties: {
+							ID: { type: 'integer' },
+							URL: { type: 'string' },
+							avatar_URL: { type: 'string' },
+							email: { type: 'boolean' },
```

Comment:
Good catch. Our APIs have a bad habit of sending down mixed types, and sometimes they aren't documented. 

In that case this would be:
```
email: { type: [ 'boolean', 'string' ] },
```

Note that if the client doesn't care about some data, we don't necessarily need to document it in the schema. Unless we set `additionalProperties: false` at a given definition level, by default additional props are valid.